config/docker/gcc-10_arm/Dockerfile

```diff
@@ -27,4 +27,5 @@ RUN apt-get update && apt-get install --no-install-recommends -y \
    libnuma-dev:armhf \
    libpopt-dev:armhf \
    libasound2-dev:armhf \
-   pkg-config:armhf
+   libasound2-dev \
```

Comment:
So is the host package needed in order to get the header files installed, and the arch-specific one for linkage?

Reply:
The host package is needed to drop a configuration file in the pkg-config directory so it knows about the library and can produce appropriate flags without error - otherwise the target -dev package should be enough.